### PR TITLE
Modified two_sum.py and two_sum_test.py files and passed two_sum tests

### PR DESCRIPTION
**Pull request by hawktalk25**

### CODE REVIEW

File: functions.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ Each student should select **one** of the following tests, implement the functio
 `twoSum` is a function which finds which two numbers in a list of numbers add up to the given target.
 
 ```py
-def twoSum(n, t):
-    for i in range(len(n)):
-        for j in range(i + 1, len(n)):
-            if n[j] == t - n[i]:
-                return [i, j]
+def twoSum(list, target):
```
Comment (MDHowden2020) [inline functions.md:10]: two_sum is more pythonic, but nice try though

Comment (MDHowden2020) [reply functions.md:10]: Also, it would be beneficial to use a list comprehension here instead

File: functions/two_sum.py [hunk 2]
```diff
@@ -1,2 +1,5 @@
-def twoSum(n, t):
-    pass
+def twoSum(list, target):
+    for i in range(len(list)):
```
Comment (MDHowden2020) [inline functions/two_sum.py:2]: Instead of nested for loops here, you could have used lambda

**Review by MDHowden2020 — COMMENTED** [review]
Overall, You did a great job Anhtagonistic, but I see room for improvement lol.

File: functions/two_sum.py (hunk 2 above)
Comment (Coolcud) [reply functions/two_sum.py:2]: I disagree with lambda, though I do agree the nested for loops should be changed.

File: functions.md [hunk 3]
```diff
@@ -7,11 +7,11 @@ Each student should select **one** of the following tests, implement the functio
...
+    for i in range(len(list)):
+        for j in range(i + 1, len(list)):
+            if list[j] == target - list[i]:
+                return [list[i], list[j]]
```
Comment (Coolcud) [inline functions.md:14]: Improve the time complexity instead of using nested for loops.

**Review by Coolcud — COMMENTED** [review]
Was fine. But do better next time.

**Pull request merged**